src/app.js

```diff
@@ -1 +1,82 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+
+console.log('Hallo daar!');
+
+// Reference naar country-list
+const countryList = document.getElementById('country-list');
+
+// Functie die een region verwacht en de juiste kleur-naam-attribuut geeft voor de naam van het land
+function countryColor (region) {
```

Comment:
Als ik jou was, zou ik switch statement gebruiken. dit is niet fout maar switch ziet er meer netjes uit:

function countryColor (region) {
    switch(region) {
        case "Africa":
         return 'africa country-name'
        break;

        case "Americas" :
        return "americas country-name"
        break;

        case "Asia": 
        return "asia country-name"
        break;

        case 'Europe':
        return 'europe country-name'
        break;

        case 'Oceania':
        return 'oceania country-name'
        break;

        default:
        return 'other country-name'            

    }
}